package ru.vcarstein.kokodi.exception

class GameFinishedException: RuntimeException("Game is already finished") {
}